### DISCUSSION TITLE
[v5-beta] I am getting a 404 at `http://localhost:3000/api/auth/callback/github?code=???` on github authentification

### DISCUSSION THREAD

**Discussion by AaronNGray** [nextauthjs/next-auth#11688] [Help] [answered]
AFAICT I have followed the instructions at :-

https://authjs.dev/getting-started/installation
https://authjs.dev/getting-started/authentication/oauth

And I am getting a 404 on callback return to :-

    http://localhost:3000/api/auth/callback/github?code=xxxxxxxxxxxxxxxxxxxx

I have made a test example repo to demonstrate the issue :-

https://github.com/AaronNGray/nextauth-js-5-beta-test

Hopefully I have made a simple mistake as I am not at all familiar with NextAuth.

**Comment by k-taro56** [comment, marked as answer]
Maybe it's fixed
https://github.com/AaronNGray/nextauth-js-5-beta-test/pull/1

**Comment by AaronNGray** [comment]
solved !
